.github/ISSUE_TEMPLATE/game-bug-report.yaml

```diff
@@ -0,0 +1,82 @@
+# Docs - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Game Emulation Bug Report
+description: Problem in a game (ie. graphical artifacts, crashes, etc.)
+title: "[GAME BUG]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Important: Read First
+
+        Please do not make support requests on GitHub. Our issue tracker is for tracking bugs and feature requests only
+        If you need help using the emulator or unsure about your issue please contact us on [discord](https://discord.gg/bFJxfftGW6).
+
+        You can also check the [Game Compatibility Repository](https://github.com/shadps4-emu/shadps4-game-compatibility) for the information about the status of the game.
+
+        Please make an effort to make sure your issue isn't already reported.
+
+        Do not create issues involving software piracy, our rules specifically prohibit this. Otherwise your issue will be closed and you will be banned in this repository.
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      options:
+        - label: I have searched for a similar issue in this repository and did not find one.
+          required: true
+        - label: I am using an official build obtained from [releases](https://github.com/shadps4-emu/shadPS4/releases) or updated one of those builds using its in-app updater.
+          required: true
+        - label: I have re-dumped the game and performed a clean install without mods.
+          required: true
+        - label: I have disabled all patches and cheats.
+          required: true
+        - label: I have all the required [system modules](https://github.com/shadps4-emu/shadps4-game-compatibility?tab=readme-ov-file#informations) installed.
+          required: true
+  - type: textarea
+    id: desc
+    attributes:
+      label: Describe the Bug
+      description: "A clear and concise description of what the bug is"
+    validations:
+      required: true
+  - type: textarea
+    id: repro
+    attributes:
+      label: Reproduction Steps
+      description: "Detailed steps to reproduce the behavior"
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: Specify OS Version
+      placeholder: "Example: Windows 11, Arch Linux, MacOS 15"
+    validations:
+      required: true
+  - type: input
+    id: cpu
+    attributes:
+      label: CPU
+      placeholder: "Example: i7-8700"
```

Comment:
Your comment made me think that it might also be worth specifying an iGPU (for example, if a person uses, say, Ryzen AI 9 HX 370), but wouldn't it be superfluous?

Reply:
> Your comment made me think that it might also be worth specifying an iGPU (for example, if a person uses, say, Ryzen AI 9 HX 370), but wouldn't it be superfluous?

Doesn't matter if it's dGPU or iGPU, it still goes into the GPU input field.

Reply:
But here I meant that the user can play not on a discrete graphics card, but on an integrated one.

Reply:
> But here I meant that the user can play not on a discrete graphics card, but on an integrated one.

And? What do you want to change in the template to accommodate for that?

Reply:
Oh, I didn't refresh the page, so my answer wasn't to you)

But if so, it's pretty hard to say.

I do not know how to add this, but it can be something like - (or iGPU, if you are not using a discrete graphics card, but a processor with powerful integrated graphics)

That's why I said it might be unnecessary, but who knows.

Reply:
And if you think about it that way, you can even remember that the user may have overclocking of the processor, memory, video card, or undervolting, which may also affect the operation of the application.

But that's if you go too far.